src/LIMITATIONS.h

```diff
@@ -0,0 +1,33 @@
+// Limitations in uncrustify are shown here:
+
+// uncrustify_types.h:
+#define MAX_OPTION_NAME_LEN    32
```

Comment:
BTW, it would be nice to use `constexpr` variables instead of macros. Maybe even put them in an `uncrustify::limits` namespace?